wallet-api/src/Wallet/Generators.hs

```diff
@@ -146,30 +148,30 @@ genValidTransaction' g f (Mockchain bc ops) = do
                     <$> (catMaybes
                         $ traverse (pubKeyTxo [bc]) . (di . fst) <$> inUTXO)
         inUTXO = take nUtxo $ Map.toList ops
-        totalVal = foldl' (Value.plus) Value.zero $ (map (txOutValue . snd) inUTXO)
+        totalVal = foldl' (+) 0 $ (map (Ada.fromValue . txOutValue . snd) inUTXO)
         di a = (a, a)
         mkSig (PubKey i) = Signature i
     genValidTransactionSpending' g f ins totalVal
 
 genValidTransactionSpending :: MonadGen m
     => Set.Set TxIn
-    -> Value
+    -> Ada
```

Comment:
Why not keep this as `Value`?

Reply:
Because this calls `splitVal :: (MonadGen m, Integral n) =>Int -> n -> m [n] ` and there we would have to change `n` to `Value` and effectively generate a linear combination of vectors

Reply:
I mean, we'd probably like to do that in the end, but seems okay to be out of scope here.

Reply:
On the other hand, all the values we have on the mockchain are Ada, and if we wanted to generate different currencies we would have to generate their monetary policy scripts as well, so even if we had a `splitVal` that works properly on `Value`s we still wouldn't have any transactions with `Value`s that are more interesting than what we generate now